lightning/kv/importer.go

```diff
@@ -85,15 +86,18 @@ func NewImporter(ctx context.Context, importServerAddr string, pdAddr string) (*
 	}
 
 	return &Importer{
-		conn:   conn,
-		cli:    kv.NewImportKVClient(conn),
-		pdAddr: pdAddr,
+		conn:             conn,
+		cli:              kv.NewImportKVClient(conn),
+		pdAddr:           pdAddr,
+		retryBackoffTime: defaultRetryBackoffTime,
 	}, nil
 }
 
 // Close the importer connection.
 func (importer *Importer) Close() {
-	importer.conn.Close()
+	if importer.conn != nil {
+		importer.conn.Close()
```

Comment:
Should we return the error to the caller?

Reply:
Added a log on error. Not going to return the error to caller since no caller cares anyway.

Reply:
OK